integrations/airflow/marquez_airflow/extractors/sql/parser.py

```diff
@@ -45,20 +44,35 @@ def _match_on(token, keywords):
     return token.match(T.Keyword, values=keywords)
 
 
-def _get_table(tokens, idx):
+def _get_tables(tokens, idx) -> Tuple[int, List[DbTableName]]:
+    def parse_ident(ident: Identifier) -> str:
```

Comment:
please add a comment on this function describing what it does

Reply:
Done.